### PR TITLE
removed remaining old components code

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Cleaned remaining code from old components](https://app.shortcut.com/oazo-apps/story/4770/clean-up-old-components-code)
  
## Changes 👷‍♀️
  New components flag was already removed and merged here - https://github.com/OasisDEX/oasis-borrow/pull/1150.
 Scope of this PR is to clean up the remaining dead code and TODOs from old components.
  
## How to test 🧪
  - Check if everything is fine.
  - Let me know if there are more pieces of code that can be removed.
  - Help me clarify if `TODO` in `GeneralManageLayout.tsx` is up to date, what can be removed?

### CODE REVIEW

File: components/vault/GeneralManageLayout.tsx [hunk 1]
```diff
@@ -54,7 +54,8 @@ export function GeneralManageLayout({
         defaultMode={VaultViewMode.Overview}
         heading={t(vaultHeadingKey, { ilk: vault.ilk, id: vault.id })}
         headline={headlineElement}
-        // TODO this prop to be removed when newComponentsEnabled wont be needed anymore
+        // TODO this prop to be removed when newComponentsEnabled wont be needed anymore.
+        // ^^ Not quite sure what can be removed. Isn't comment above is outdated? ~ŁW
```
Comment (George-field) [inline components/vault/GeneralManageLayout.tsx:58]: Yeah this looks like an outdated comment, may be best to just remove it to avoid future confusion.

**Review by George-field — CHANGES_REQUESTED** [review]
All good, just think we need to remove the comment you mentioned as it is outdated and will cause confusion :)

File: components/vault/GeneralManageLayout.tsx (hunk 1 above)
Comment (johnnieskywalker) [reply components/vault/GeneralManageLayout.tsx:58]: Ok let's remove it

**Review by George-field — APPROVED** [review]
All good :)

**Pull request merged**